test/test.py

```diff
@@ -456,6 +476,10 @@ def _assert_bundle(self, uuid, version, manifest, metadata_files,
             self.assertEqual(slice_thickness,
                              [s.slice_thickness for s in bundle.entities.values() if isinstance(s, ImagedSpecimen)])
 
+        if content_description is not None:
+            self.assertSetEqual(content_description,
+                                set(chain(*(file.content_description for file in bundle.files.values()))))
```

Comment:
`chain.from_iterables` is my default solution because it is lazy and therefore more efficient for large inputs. `*` is eager and creates a tuple, a potentially large one.